android/build.gradle

```diff
@@ -23,7 +23,7 @@ allprojects {
 }
 
 subprojects { afterEvaluate { android { 
-    // compileSdkVersion 34
+     compileSdkVersion 34
```

Comment:
we use flutter based SDK... as we use a recent flutter SDK, the android sdk is also recent. this line isn't needed IMHO